config/environments/production.rb

```diff
@@ -49,7 +49,13 @@
 
   # Use a different logger for distributed setups.
   # config.logger = ActiveSupport::TaggedLogging.new(SyslogLogger.new)
-  config.logger = Logger.new(STDOUT)
+  path = if ENV['LOG_DIR_PATH']
+    "#{ENV['LOG_DIR_PATH']}/#{Rails.env}.log"
```

Comment:
Use 2 (not -5) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.